spring-cloud-dataflow-shell/src/test/java/org/springframework/cloud/dataflow/shell/command/TaskCommandTests.java

```diff
@@ -35,4 +35,16 @@ public void testCreateTask() throws InterruptedException {
 		String taskName = generateUniqueName();
 		task().create(taskName, "foobar");
 	}
+
+	@Test
```

Comment:
The way those tests are architectured feels really weird IMHO: the `@Test` methods don't assert anything, rather assertions are in `TaskCommandTemplate` ?? (if I understand correctly)